flask_lambda.py

```diff
@@ -98,7 +103,7 @@ def __call__(self, event, context):
 
         response = LambdaResponse()
 
-        body = next(self.wsgi_app(
+        body = b''.join(self.wsgi_app(
```

Comment:
This change here fixed the issue we've been having for days.

204 response codes in Werkzeug break `next` as the iterator that's returned is wrong somehow. Doing this instead works just fine.

Reply:
Awesome! Glad to hear it. This had me stumped for a while too.

Reply:
Thanks for this! Fixed an issue I was having with OPTIONS requests.